import_export/widgets.py

```diff
@@ -71,15 +71,22 @@ def render(self, value, obj=None):
 class FloatWidget(NumberWidget):
     """
     Widget for converting floats fields.
+    
+    Takes optional ``coerce_to_string`` parameter, set to ``True`` the 
+    :meth:`~import_export.widgets.Widget.render` method will return a string
+    else it will return a value`
```

Comment:
I suggest add this to `NumberWidget` so that it doesn't need to be duplicated in `DecimalWidget` and `FloatWidget`.